lua/dap.lua

```diff
@@ -325,6 +333,75 @@ local function jump_to_frame(frame, preserve_focus_hint)
   end
 end
 
+--interface ExceptionInfoRequest extends Request {
```

Comment:
So far we didn't add the request structure definitions, not sure if we should start doing that now - and if so, maybe closer to the structure definitions where we use them?

Reply:
We could also replace it by a link to the specification.

Reply:
Added a short documentation with links to specification at the points we use the request/responses